src/PolyhedralGeometry/Polyhedron/constructors.jl

```diff
@@ -4,11 +4,17 @@
 ###############################################################################
 ###############################################################################
 
-struct Polyhedron{T} #a real polymake polyhedron
+struct Polyhedron{T<:scalar_types} #a real polymake polyhedron
     pm_polytope::Polymake.BigObject
-    
+
     # only allowing scalar_types;
     # can be improved by testing if the template type of the `BigObject` corresponds to `T`
+
+    """
```

Comment:
```suggestion
    @doc Markdown.doc"""
```
This might help the failing docs build?

Otherwise the signature in the docs file might need the type parameter, but I am slightly confused why it did work previously, as the docs [refer](https://github.com/oscar-system/Oscar.jl/blob/c6704863786a39a724ded14b6ff95e5293158a0d/docs/src/PolyhedralGeometry/Polyhedra/polymake.md?plain=1#L33) to the non-typed construction without a `@doc` block.

Reply:
We had similar issues with the tropical stuff, e.g. in 05b9ade114555ed53afb74874f922952b2a29b9d we had to add this `where` statement. It later became clear that the `where` statements could be left out in much of the toric code to make it more generic, which is why the docs look nice again. ;)

Reply:
The removed `@docs Markdown.doc` is not the problem. Rather something else is wonky. In `docs/src/PolyhedralGeometry/Polyhedra/polymake.md` it does this:
````
```@docs
Polyhedron(::Polymake.BigObject)
```
````

But so far, the docstring this pulls in is that for `Polyhedron{T}(P::Polymake.BigObject) where T<:scalar_types` -- which by itself is IMHO surprising, rather it should always have complained that there is no docstring for that constructor.

The question I can't answer is what you want:
1. should there be a docstring for `Polyhedron(::Polymake.BigObject)`, i.e., without the type parameter, and that be linked?
2. should the `.md` file be fixed to reference the docstring for `Polyhedron{T}` ?

Reply:
Adding the suggested ```@doc ...``` does fix the docs-build on commit https://github.com/oscar-system/Oscar.jl/commit/4d993df85265cce4adbb8f72b0fe8b25eeb6269f, without it I get the `no docs found`, adding it fixes the build.

I don't claim that this is correct and I also mentioned in my previous comment that the non-typed docs code pulls in the typed constructor for some reason. I guess the docs file was missed in one of the recent updates that added the types (which is not that surprising given it still seems to build ...).

The type-parameter is optional in that constructor, it will be determined from the type-string of the `BigObject` and it is mostly for internal use / power-users who need to deal with the polymake objects directly. So I am not sure about the best way to write that in the signature.